core/domain/summary_services_test.py

```diff
@@ -134,3 +135,51 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
         self.assertIn('last_updated_msec', displayable_summaries[0])
         self.assertDictContainsSubset(expected_summary,
                                       displayable_summaries[0])
+
+
+class FeaturedExplorationDisplayableSummaries(
+        test_utils.GenericTestBase):
+    """Test functions for getting displayable
+     featured exploration summary dicts."""
+
+    ALBERT_NAME = 'albert'
+    ALBERT_EMAIL = 'albert@example.com'
+
+    EXP_ID_1 = 'eid1'
+    EXP_ID_2 = 'eid2'
+
+    def setUp(self):
+        """Populate the database of explorations and their summaries.
+
+        The sequence of events is:
```

Comment:
This sequence does not match the order of operations in setUp(). It doesn't affect the functionality, but I would suggest fixing it in order to avoid unnecessary confusion.

You could also completely omit this part of the docstring; it's pretty self-evident.